physics/cu_gf_deep.F90

```diff
@@ -560,6 +560,7 @@ subroutine cu_gf_deep_run(        &
          c1d(i,:)= 0. !c1 ! 0. ! c1 ! max(.003,c1+float(csum(i))*.0001)
          entr_rate(i)=7.e-5 - min(20.,float(csum(i))) * 3.e-6
          if(xland1(i) == 0)entr_rate(i)=7.e-5
+         if(dx(i)<6500.) entr_rate(i)=2.e-4
```

Comment:
Since the test for `dx(i)<6500.` shows up multiple times, and I'm guessing that the 6500 number is tunable, consider changing this to a variable/parameter that is set at the top of the subroutine.

Reply:
@grantfirl Thanks for your suggestions. Yes, we put this number for both RRFS and future HAFS applications. I will think how to properly deal with this number.